assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -3298,7 +3298,12 @@ protected String build_instructor_grade_submission_context(VelocityPortlet portl
                     for (String userId : users.keySet()) {
                         String userGrade = (String) state.getAttribute(GRADE_SUBMISSION_GRADE + "_" + userId);
                         if (userGrade != null) {
-                            grades.put(userId, assignmentService.getGradeDisplay(userGrade, gradeType, a.getScaleFactor()));
+                            if (a.getTypeOfGrade() == SCORE_GRADE_TYPE) {
```

Comment:
AssignmentServiceImpl.getGradeDisplay is able to handle displaying any grade type there is no need to perform this check here